test/parallel/test-http-expect-continue.js

```diff
@@ -20,70 +20,63 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
-require('../common');
+const common = require('../common');
 const assert = require('assert');
 const http = require('http');
 
-let outstanding_reqs = 0;
 const test_req_body = 'some stuff...\n';
 const test_res_body = 'other stuff!\n';
 let sent_continue = false;
 let got_continue = false;
 
-function handler(req, res) {
-  assert.strictEqual(sent_continue, true,
-                     'Full response sent before 100 Continue');
+const handler = common.mustCall((req, res) => {
+  assert.ok(sent_continue, 'Full response sent before 100 Continue');
   console.error('Server sending full response...');
   res.writeHead(200, {
     'Content-Type': 'text/plain',
     'ABCD': '1'
   });
   res.end(test_res_body);
-}
+});
 
-const server = http.createServer(handler);
-server.on('checkContinue', function(req, res) {
+const server = http.createServer();
```

Comment:
`handler` is no longer registered as a 'request' listener?

Reply:
Actually, given the description in the API docs for [`checkContinue`](https://nodejs.org/dist/latest-v9.x/docs/api/http.html#http_event_checkcontinue) it probably makes more sense to register a `common.mustNotCall` request listener.